### ISSUE TITLE
Update developer installation instructions

### ISSUE THREAD

**Issue by jhamrick** [jupyter/nbgrader#862]
So that when installing the extensions, all the commands are in the same code block.

**Comment by jzf2101** [comment]
@fenwickipedia also might be something to consider

**Comment by fenwickipedia** [comment]
Okay this looks good, I'm working on it. Thanks!

**Comment by jhamrick** [comment]
Reopening this issue because #911 didn't quite address what I had in mind, though this is totally my bad for (1) not explaining well enough in the issue description and (2) not being around to review. More specifically, in the developer installation instructions [here](https://github.com/jupyter/nbgrader/blob/master/nbgrader/docs/source/contributor_guide/installation_developer.rst#installing-notebook-extensions) there are two separate code blocks containing required pieces of the installation process, one that looks like this:

```
# The nbextensions are JavaScript/HTML/CSS so they require
# separate installation and enabling.
jupyter nbextension install --symlink --sys-prefix --py nbgrader
jupyter nbextension enable --sys-prefix --py nbgrader
```

and one which looks like this:

```
# The serverextension is a Python module inside nbgrader, so only an
# enable step is needed.
jupyter serverextension enable --sys-prefix --py nbgrader
```

These should ideally be merged so they are in the same code block (as in the user guide installation instructions) to make it easier for people to just copy and paste everything in one go.

**Comment by jhamrick** [comment]
Fixed by #989
